lbry/lbry/extras/daemon/Components.py

```diff
@@ -224,6 +208,7 @@ def _check_header_file_integrity(self):
                 log.error("failed to fetch headers from s3: %s", err)
             finally:
                 self._downloading_headers = False
```

Comment:
I think this code would be easier to read if it was (settings True/False on is_download_headers at the same indent level is easy on the eyes):
```
        if await self.should_download_headers_from_s3():
            try:
                self.is_downloading_headers = True
                await self.fetch_headers_from_s3()
            except Exception as err:
                log.error("failed to fetch headers from s3: %s", err)
            finally:
                self._downloading_headers = False
```